src/distribute.js

```diff
@@ -0,0 +1,186 @@
+const CATEGORY_SOLIDITY_ENUM_SEED_CONTRIBUTORS = 1;
+const CATEGORY_SOLIDITY_ENUM_CONSULTANTS = 2;
+const CATEGORY_SOLIDITY_ENUM_OTHER = 3;
+const CATEGORY_SOLIDITY_ENUM_FUTURE_OFFERING = 4;
+
+const parseContractAbi = (contractPath) => {
+    return JSON.parse(fs.readFileSync(contractPath, 'utf8')).abi;
+};
+
+const pht2Wei = (pht) => {
+    return web3.utils.toBN(web3.utils.toWei(pht.toString(), 'ether'));
+};
+
+const waitFor = (waitInSeconds) => {
+    return new Promise((resolve) => {
+        setTimeout(resolve, waitInSeconds * 1000);
+    });
+};
+
+const initContract = () => {
+    return web3.eth.Contract(
+        parseContractAbi(CONTRACT_PATH),
+        CONTRACT_ADDRESS,
+        {
+            defaultGasPrice: '500000000000'
+        }
+    )
+};
+
+const isProject = (csvCategory) => {
+    return !(isPrivateSale(csvCategory) || isPublicSale(csvCategory))
+};
+
+const isPrivateSale = (csvCategory) => {
+    return csvCategory === CATEGORY_CSV_SALE_PRIVATE;
+};
+
+const isPublicSale = (csvCategory) => {
+    return csvCategory === CATEGORY_CSV_SALE_PUBLIC;
+};
+
+const calculateDepositValue = (purchasedBn, bonusBn) => {
+    return purchasedBn.add(bonusBn);
+};
+
+const logDeposit = (wallet, to, value, category) => {
+    console.log("Depositing:");
+    console.log(`   from: ${wallet}`);
+    console.log(`   to: ${to}`);
+    console.log(`   value: ${value}`);
+    console.log(`   category: ${category}`);
+};
+
+const csvCategoryToSolidityEnumValue = (csvCategory) => {
+    switch (csvCategory) {
+        case CATEGORY_CSV_PROJECT_TEAM:
+            return CATEGORY_SOLIDITY_ENUM_TEAM;
+        case CATEGORY_CSV_PROJECT_SEED_CONTRIBUTORS:
+            return CATEGORY_SOLIDITY_ENUM_SEED_CONTRIBUTORS;
+        case CATEGORY_CSV_PROJECT_CONSULTANTS:
+            return CATEGORY_SOLIDITY_ENUM_CONSULTANTS;
+        case CATEGORY_CSV_PROJECT_OTHER:
+            return CATEGORY_SOLIDITY_ENUM_OTHER;
+        case CATEGORY_CSV_PROJECT_FUTURE_OFFERING:
+            return CATEGORY_SOLIDITY_ENUM_FUTURE_OFFERING;
+        case CATEGORY_CSV_SALE_PRIVATE:
+            throw 'CSV_CATEGORY_PRIVATE_SALE_HAS_NO_SOLIDITY_ENUM_VALUE';
+        case CATEGORY_CSV_SALE_PUBLIC:
+            throw 'CSV_CATEGORY_PUBLIC_SALE_HAS_NO_SOLIDITY_ENUM_VALUE';
+        default:
+            throw 'CSV_CATEGORY_UNKNOWN';
+    }
+};
+
+const transfer1PhtToEveryDistributionAddress = async (data) => {
+    data.forEach(async function (distribution) {
+        console.log(`Transferring 1 PHT to ${distribution.to}...`);
+
+        await web3.eth.sendTransaction({
+            from: WALLET_PROJECT,
+            to: distribution.to,
+            value: pht2Wei('1'),
+            gas: 21000
+        });
+    });
+
+    console.log("All distribution accounts were topped-up!");
+};
+
+csv()
+.fromFile(CSV_FILEPATH)
+.then(async (data) => {
+    const txReceipts = [];
+    const contract = initContract();
+
+    try {
+        await transfer1PhtToEveryDistributionAddress(data);
```

Comment:
[MEDIUM] I would rather include an additional "after check" which verify every account balance is `1 PHT`